hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/OMDBInsightEndpoint.java

```diff
@@ -255,15 +253,39 @@ public Response getOpenKeyInfo(
         break;
       }
     }
-    // Populate the keysSummary map
-    createKeysSummaryForOpenKey(keysSummary);
-
-    openKeyInsightInfo.setKeysSummary(keysSummary);
```

Comment:
@ArafatKhan2198 Thanks for working on this patch, If we have removed keySummary from this API endpoint, then kindly update API doc as well.

Reply:
Thanks for the comment @devmadhuu 
I have made the changes!